itest/starboard_cli_test.go

```diff
@@ -680,11 +678,6 @@ var _ = Describe("Starboard CLI", func() {
 		// TODO run by the BeforeEach callback fails when it attempts to create Kubernetes objects in the
```

Comment:
We can remove these comments as well. They were related to deleting the namespace.